pygameMenu/widgets/colorinput.py

```diff
@@ -0,0 +1,417 @@
+# coding=utf-8
+"""
+pygame-menu
+https://github.com/ppizarror/pygame-menu
+
+COLOR INPUT
+Color input class, Widget created in top of TextInput that provides a textbox
+for entering and previewing colors in RGB and HEX format.
+
+License:
+-------------------------------------------------------------------------------
+The MIT License (MIT)
+Copyright 2017-2020 Pablo Pizarro R. @ppizarror
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the Software
+is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-------------------------------------------------------------------------------
+"""
+
+from pygameMenu.widgets.textinput import TextInput
+import pygameMenu.locals as _locals
+import pygame as _pygame
+
+_TYPE_HEX = 'hex'
+_TYPE_RGB = 'rgb'
+
+
+# noinspection PyTypeChecker
+class ColorInput(TextInput):
+    """
+    Color input widget.
+    """
+
+    def __init__(self,
+                 label='',
+                 colorinput_id='',
+                 color_type=_TYPE_RGB,
+                 input_comma=',',
```

Comment:
input_sep ?

Reply:
you mean to change the name of the parameter?

Reply:
yes, because is not necessary a comma ;)
however there is no matter, it's working like that.

Reply:
ah true, so I failed in both testing (assumed that there's a comma) and the of the parameters. I'll change this now.